src/element/mutateElement.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  MutableExcalidrawElement,
+  MutableExcalidrawTextElement,
+} from "./types";
+
+export function mutateElement(
```

Comment:
Also need a comment here otherwise it’s very confusing why you’d want this function